src/app/components/Header/index.tsx

```diff
@@ -49,50 +49,88 @@ export function Header() {
     );
   };
   const pages = [
-    { to: '/buy-sov', title: t(translations.mainMenu.buySov), exact: true },
+    {
+      to: '/buy-sov',
+      title: t(translations.mainMenu.buySov),
+      exact: true,
+      dataAttribute: 'header-trade-link-buySov',
+    },
     {
       to: '/swap',
       title: t(translations.mainMenu.swap),
+      dataAttribute: 'header-trade-link-swap',
     },
     {
       to: '/spot',
       title: t(translations.mainMenu.spotTrade),
+      dataAttribute: 'header-trade-link-spot',
     },
     {
       to: '/trade',
       title: t(translations.mainMenu.marginTrade),
+      dataAttribute: 'header-trade-link-margin',
     },
     {
       to: '/lend',
       title: t(translations.mainMenu.lend),
+      dataAttribute: 'header-finance-link-lend',
     },
     {
       to: '/borrow',
       title: t(translations.mainMenu.borrow),
+      dataAttribute: 'header-finance-link-borrow',
+    },
+    {
+      to: '/yield-farm',
+      title: t(translations.mainMenu.yieldFarm),
+      dataAttribute: 'header-finance-link-yieldFarm',
     },
-    { to: '/yield-farm', title: t(translations.mainMenu.yieldFarm) },
     {
       to: 'https://bitocracy.sovryn.app',
       title: t(translations.mainMenu.governance),
+      dataAttribute: 'header-bitocracy-link-governance',
+    },
+    {
+      to: '/stake',
+      title: t(translations.mainMenu.staking),
+      dataAttribute: 'header-bitocracy-link-stake',
+    },
+    {
+      to: '/reward',
+      title: t(translations.mainMenu.reward),
+      dataAttribute: 'header-bitocracy-link-rewards',
```

Comment:
```suggestion
      dataAttribute: 'header-link-rewards',
```

Reply:
thank you for the review :) I am going to finish the naming